pkg/utils/utils.go

```diff
@@ -169,6 +176,14 @@ func configSriovDevice(iface *sriovnetworkv1.Interface, ifaceStatus *sriovnetwor
 		if err != nil {
 			glog.Warningf("configSriovDevice(): unable to parse VFs for device %+v %q", iface.PciAddress, err)
 		}
+		for _, vf := range vfs {
+			if err := Unbind(vf); err != nil{
+				glog.Warningf("configSriovDevice(): fail to unbind driver for device %s", vf)
+			}
+		}
+		if err := ChangeNicSriovMode(iface.PciAddress, iface.SriovMode); err != nil {
```

Comment:
After moving to switchdev mode you need to set ethtool -K <pf interface> hw-tc-offload on